### ISSUE TITLE
Add h tags to Getting Started page (Steps 4-5)

### ISSUE THREAD

**Issue by SAUMILDHANKAR** [hackforla/website#2953]
### Overview
As a developer, I need to replace p tags with h tags on the headings of [Getting Started page](https://www.hackforla.org/getting-started) (Steps 4-5) so it becomes WCAG compliant

### Action Items
- [x] Review [Getting Started html](https://github.com/hackforla/website/blob/gh-pages/pages/getting-started.html) file
- [x] In line 166, change the p tag to an appropriate h tag: https://github.com/hackforla/website/blob/96f02c4a8446f615741c881d823b94619738a708/pages/getting-started.html#L166
- [x] In line 201, change the p tag to an appropriate h tag:
https://github.com/hackforla/website/blob/96f02c4a8446f615741c881d823b94619738a708/pages/getting-started.html#L201
- [x] Ensure h tags (h1, h2, h3 etc.) reflect the correct hierarchy of the page for screen readers
- [x] Make sure the [Getting started page](https://www.hackforla.org/getting-started) still looks the same.

### Resources/Instructions
- [Getting Started page](https://www.hackforla.org/getting-started)
- [Getting Started html](https://github.com/hackforla/website/blob/gh-pages/pages/getting-started.html) file
- [Headings in Page Structure Tutorial](https://www.w3.org/WAI/tutorials/page-structure/headings/)
- [WCAG guidelines](https://www.w3.org/WAI/standards-guidelines/wcag/)
- Related issue #2952

**Comment by SAUMILDHANKAR** [comment]
Moving the issue for reprioritization as per this [comment](https://github.com/hackforla/website/pull/3116#issuecomment-1126565335)

**Comment by arpitapandya** [comment]
Availability for this week: 6hrs
My estimated ETA for completing this issue: 1.5hr

**Comment by arpitapandya** [comment]
- Time spent: 6hrs to complete
- Availability: 6hrs for next week
- Blockers: None for this issue

**Comment by arpitapandya** [comment]
@JessicaLucindaCheng
- Found a bug which can be a separate new issue to work on(or it's already in issues list) in [Getting Started html](https://github.com/hackforla/website/blob/gh-pages/pages/getting-started.html),
- At present Step 3 and Step 4 in [Getting Started page](https://www.hackforla.org/getting-started) isn't pointing to correct card.
- changes required for the link Step 3 and Step 4
- move line 27 and 28 to line 33 and 34
- move line 33 and 34 to line 27 and 28
![getting-started-page-steps](https://user-images.githubusercontent.com/66761308/179373803-7a30af0a-cc8d-4627-8cea-e3fd9105df27.png)

**Comment by JessicaLucindaCheng** [comment]
@arpitapandya 
Thanks for bringing this up! It's already in the [issues spreadsheet] (https://docs.google.com/spreadsheets/d/1qEiDFUKPCVH3JNzGQSKl_HMl9vqNv8Z9gGrvw9MyYvU/edit#gid=772902542) as row #19. It actually should be the larger, detailed cards 3 and 4 should be switched so it matches the header order as per the Figma design.

Also, please leave your issue in the In Progress column. When your pull request is merged, there is automation that will close and move your issue to another column. I am going to go ahead and move this issue back to In Progress.
